vars/parallelMultiArchTest.groovy

```diff
@@ -24,10 +26,10 @@ def call(List<String> arches, Boolean runOnSlave, Boolean installAnsible, Closur
   parallelizeTasks(
     parallelTasks,
     { params ->
-      String arch = new String(params.arch)
+      String arch = params.arch
```

Comment:
I'd like to test this in a multiple arch scenario since weird things happen with parallel tasks in the groovy sandbox.

Reply:
good idea

Reply:
I did go back and test, and this works

Reply:
:+1: